.github/workflows/ci.yml

```diff
@@ -189,9 +189,9 @@ jobs:
         with:
           push: true
           platforms: linux/amd64,linux/arm64
-          labels: "VERSION=${GITHUB_REF#refs/*/}"
+          labels: "VERSION=${{  github.ref_name }}"
           tags: |
-            quay.io/team-helium/miner:gateway-${GITHUB_REF#refs/*/}
+            quay.io/team-helium/miner:gateway-${{  github.ref_name }}
```

Comment:
Either this needs quotes, or the quotes on line 177 are unnecessary. Do you have any idea?

Reply:
But we know the [test-images](https://quay.io/repository/team-helium/test-images?tab=tags&tag=latest) build worked, so maybe add quotes regardless?

Reply:
i think with the list setup it doesnt need quote:
```
tags: |
```

but in the other setting it does. 

let me see if i can find any docs to confirm that...

Reply:
@JayKickliter looking at these links:

https://github.com/docker/build-push-action
https://docs.github.com/en/actions/learn-github-actions/contexts 

it looks like quotes aren't needed in either setting

Reply:
How about temporarily moving 
```
          tags: |
            quay.io/team-helium/miner:gateway-${{  github.ref_name }}
            quay.io/team-helium/miner:gateway-latest
```
to the test-images section above to test? You can push and see if the two tags go to the test-images repo

Reply:
How about temporarily moving 
```
          tags: |
            quay.io/team-helium/miner:gateway-${{  github.ref_name }}
            quay.io/team-helium/miner:gateway-latest
```
to the test-images section above to test? You can push and see if the two tags go to the test-images repo

Reply:
actually @JayKickliter I don't think that will work if i do it as I don't have write access on the repo I think the repository secrets aren't available. As the quay.io pushes to test-images always fail with:

```
Run docker/login-action@v2
Error: Username and password required
```

Reply:
@JayKickliter I just tried it here and it worked fine without quotes:
https://github.com/NebraLtd/hm-diag/pull/547

Reply:
Thanks